.vsts-pipelines/builds/ci-official.yml

```diff
@@ -20,6 +20,13 @@ phases:
       displayName: 'Run compliance check'
       inputs:
         targetType: F
+    # Detect OSS Components in use in the product. Only needs to run on one OS in the matrix.
+    - task: ms.vss-governance-buildtask.governance-build-task-component-detection.ComponentGovernanceComponentDetection@0
+      displayName: 'Component Detection'
+      inputs:
+        # This funky GUID represents the product "ASP.NET and EF Core"
+        governanceProduct: 'c641993b-8380-e811-80c3-0004ffb4789e'
```

Comment:
@Eilon do you want ASP.NET SignalR OSS registrations to go to the same product or do you want me to create a different one?

Reply:
Everything into 1 please!

Reply:
:+1: